src/apps/containers/components/ContainerSearch.tsx

```diff
@@ -100,7 +100,7 @@ const ContainerSearch: React.FC<Props> = ({
           />
         }
         label={
-          showOnlyVerified ? "Showing verified only" : "Showing all results"
+          showOnlyVerified ? "Showing verified only" : "Show Verified Only"
```

Comment:
As per the description you need to add identifiers for which components you are viewing, eg show verified dictionaries.

Reply:
Thank you @hadijahkyampeire for the review, I have tried adding identifiers, will you please have a look?